pom.xml

```diff
@@ -100,6 +100,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
Hey can you post in the group chat a link to the documentation for Guava and explain what it can do & what you are using it for?

I think it would be helpful for me and the rest of the team.

Reply:
Understood o7

Reply:
Done.

Reply:
Done removing the dependency.